### DISCUSSION TITLE
Welcome to contribution-streak-maintainer Discussions!

### DISCUSSION THREAD

**Discussion by EliasAfara** [devful/contribution-streak-maintainer#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by God-starter** [comment]
Does it still work?

**Reply by EliasAfara** [reply]
Yes

**Comment by iamrikie** [comment]
After running the workflow manually, it only shows a commit has been made to 'AUTOMATED_CONTRIBUTIONS.md' but won't add to the contribution chart on the profile(the green boxes that show daily contributions). Is that how it's supposed to work?
